config/v1/types.go

```diff
@@ -137,6 +137,19 @@ type RemoteConnectionInfo struct {
 	CertInfo `json:",inline"`
 }
 
+type AdmissionConfig struct {
+	AdmissionPluginConfig map[string]AdmissionPluginConfig `json:"admissionPluginConfig"`
+
+	// forceEnableAdmissionPlugins is a list of admission plugins that must be on in addition to the default list.
+	// Some admission plugins are disabled by default, but certain configurations require them.  This is fairly uncommon
+	// and can result in performance penalties and unexpected behavior.
+	ForceEnableAdmissionPlugins []string `json:"forceEnableAdmissionPlugins"`
```

Comment:
do we really need the `force` prefix?

Also we could drop the admission prefix in all fields.